helpers/deprecated/pick.js

```diff
@@ -1,13 +1,18 @@
 'use strict';
 
-const _ = require('lodash');
-
 const factory = () => {
     /**
      * @deprecate
      */
-    return function() {
-        return _.pick.apply(null, arguments);
+    return function(obj, ...args) {
+        const pickedElements = {};
+
+        args.forEach(e => {
+            if (obj[e]) {
```

Comment:
You should probably use `hasOwnProperty` here in case the value is falsy.

Reply:
ok will do.